_posts/2023-05-26-dftf-new.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "New Repo: DFTF"
+categories: new-repo
+---
+
+[DFTF](https://github.com/LLNL/dftf), or Drink From The Firehose, is a Python program that subscribes to Redfish events on Cray/HPE hardware and republishes them to topics in Kafka. In an attempt to tame the "firehose" of information from CrayTelmetry, DFTF drops any repeated metrics so only the most recent value for each unique metric is maintained. In effect, this usually means values are reported roughly every five seconds rather than every second. See the `example.conf` file for example configuration.
```

Comment:
You also might want to change the `example.conf` file into a link? https://github.com/LLNL/dftf/blob/main/example.conf